README.md

```diff
@@ -27,7 +27,18 @@ We can add a nif validation to the myFormValues
 import { nif } from '@lemoncode/fonk-nif-validator';
 
 const validationSchema = {
-  price: [nif.validator],
```

Comment:
It's my fault but the `field` property is missing in validationSchema:

```
const validationSchema = {
  field: {
     providerCIF....
  }
}
```
I fixed it in other repositories

Reply:
Done